README.md

```diff
@@ -48,7 +48,7 @@ Both GCC 4.8 and Clang 3.4 meet this requirement.
 To install on a Debian or Ubuntu system, first install the prerequisites:
 
 ```sh
-sudo apt-get install autoconf automake libtool make g++ pkg-config libboost-dev \
+sudo apt-get install make cmake g++ pkg-config libboost-dev \
```

Comment:
The Find* CMake modules can use pkg-config to figure out where the stuff is. None of the modules used here does this (yet). So either way is fine, with or without pkg-config, but it should be consistent between the different platforms IMHO.